packages/preview/diagraph/0.1.0/README.MD

```diff
@@ -0,0 +1,130 @@
+# diagraph
+
+A simple graphviz binding for typst using the new webassembly plugin system
+
+## Usage
+
+This plugin is quite simple to use, you just need to import it:
+
+```typ
+#import "@preview/diagraph:0.1.0": *
+```
+
+It allows you to render a graphviz dot string to a svg image:
+
+```typ
+#render("digraph { a -> b }")
+```
+
+For a simpler usage, you can setup a simple show rule:
+
+```typ
+#show: gaph-render
```

Comment:
it's still wrong here

Reply:
No, graph render is meant to be used as a show rule. You can simply do this:
```typst
#import "@preview/diagraph:0.1.0": *
#show: gaph-render.with(height: 15em)

```dotrender
digraph {
	rankdir=LR;
	A -> B
	B -> B
}
```

Reply:
I didn't mean that it's not a show rule, I just meant that it's still "gaph" instead of "graph". But I see that you have renamed it to something different now.

Reply:
Ok, I didn't spot that this was a typo. I renamed it because, in other packages, the show functions contain `rule` in their name.